src/service/components/sound.js

```diff
@@ -2,23 +2,26 @@
 //
 // SPDX-License-Identifier: GPL-2.0-or-later
 
-'use strict';
+import Gdk from 'gi://Gdk';
+import Gio from 'gi://Gio';
+import GLib from 'gi://GLib';
 
-const Gdk = imports.gi.Gdk;
-const Gio = imports.gi.Gio;
-const GLib = imports.gi.GLib;
+let GSound = null;
+try {
+    GSound = (await import('gi://GSound')).default;
+} catch (e) {}
 
 
 /*
  * Used to ensure 'audible-bell' is enabled for fallback
  */
-const WM_SETTINGS = new Gio.Settings({
+/* const WM_SETTINGS = new Gio.Settings({
```

Comment:
Ia this variable not used anywhere?

Reply:
Right, these comments are because ESLint complains about unused global functions/variables which aren't explicitly exported now since we're on ESM

I can drop all 3 of these if you want, but yeah none of these are referenced anywhere